crafts.lua

```diff
@@ -39,13 +41,17 @@ local translated_colors = {
 	pink       = S("Pink"),
 }
 
+
+
+
 local function get_dye_color(name)
 	local color
 	if has_unifieddyes then
 		color = unifieddyes.get_color_from_dye_name(name)
 	end
 	if not color then
-		color = string.match(name, "^dye:(.+)$")
+		color = string.match(name, dye_prefix_pattern_universal.."(.+)$")
+		string.gsub(name, dye_suffix_pattern_farlands, "")
```

Comment:
Shouldn't this continue using `color`? Otherwise, line 54 wouldn't do anything.
```suggestion
		string.gsub(color, dye_suffix_pattern_farlands, "")
```

Reply:
Oh thanks for spotting that, without that line farlands wouldn't have color support